docs/contivctl.md

```diff
@@ -0,0 +1,226 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+Command can be found under $GOPATH/bin directory (/opt/gopath/bin in vagrant VMs) after a build. 
+Currently, contivctl needs to be run on the same host where netmaster is running
+
+## Usage
+
+```
+vagrant@netplugin-node1:~$ contivctl --help
+usage: contivctl [-h] [--version] {policy,rule,group,network} ...
+
+positional arguments:
+  {policy,rule,group,network}
+    policy              Policy operations
+    rule                Rule add/delete
+    group               Endpoint group operations
+    network             Network operations
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --version             show program's version number and exit
+```
+
+### Creating networks
```

Comment:
May be this section should be called just `Networks` as it walk through examples of creation, listing and deletion.

Same goes for section on policies, rules an groups.

Also wondering if these sections shall start with one-liner descriptions of what a network, group, policy and rule mean. That will give reader some insight into the concepts. We can also put cross-reference to more detailed write-ups in future for interested readers.